docs/content/latest/develop/graphql/hasura.md

```diff
@@ -11,21 +11,23 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Integrate the [Hasura GraphQL engine](https://hasura.io) with YugabyteDB to use GraphQL on your YugabyteDB databases and applications.
+Use the [Hasura GraphQL Engine](https://hasura.io) with YugabyteDB to use GraphQL with your YugabyteDB databases and applications.
 
-Follow the steps below to learn how easily you can begin using the Hasura GraphQL engine with YugabyteDB. For details on using the Hasura GraphQL engine, see the [Hasura GraphQL engine documentation](https://docs.hasura.io).
+Follow the steps below to learn how easily you can begin using the Hasura GraphQL Engine with YugabyteDB. For details on using the Hasura GraphQL engine, see the [Hasura GraphQL engine documentation](https://docs.hasura.io).
 
 ## Before you begin
 
 ### Install and start YugabyteDB
 
-Before starting and running YugabyteDB with Hasura, you need to add the YugabyteDB environment variable `YB_SUPPRESS_UNSUPPORTED_ERROR=1`. Setting the value to 1 suppresses unsupported error exceptions and raise only warnings. To set the environment variable , run the following command.
+To use the Hasura GraphQL Engine with YugabyteDB, you need to set the `yb-tserver` option `--ysql_suppress_unsupported_error` to `true` so that errors on the use of unsupported SQL statements and return only warnings instead.
```

Comment:
```suggestion
To use the Hasura GraphQL Engine with YugabyteDB, you need to set the `yb-tserver` option `--ysql_suppress_unsupported_error` to `true` so that errors on the use of unsupported SQL statements are suppressed and only raise warnings instead.
```